tests/hash_files/README.md

```diff
@@ -1,8 +1,7 @@
 These are files that are used to test the internal hashing function of pytest-
 workflow.
 
-The linked license file was used because it was bigger than 8192 bytes and 
-therefore truly tests the function which uses a 8192 bytes block size.
+A link to the license is present because it contains ample text to check and
+we do not need to distribute another file in the git repo.
 
-The gzipped file is also bigger than 8192 bytes and tests the function on 
-binary data.
```

Comment:
it seems like the original text here is no longer accurate, the current block size for the md5 util here is 64K.

Reply:
Good point. I should still make sure that the sum addition in file_md5sum works correctly. I can do so by setting the block_size lower in the function where I test file_md5sum.
This entire README can be removed then.

Reply:
Ah upon further reading, please simply move this to the diff files repo.
You can also delete both READMEs and create two different files in the diff folder with a little less dry content to diff if that is your cup of tea. Feel free to be creative.